<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbAgregar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLista">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVerStock">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver stock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
